### PR TITLE
Simplified assignment

### PR DESCRIPTION
**Pull request by dryanmas**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -163,6 +139,84 @@ function processDuplicateFree(/* CODE HERE ONLY AFTER COMPLETING ALL OTHER TASKS
 /////////////// HIGHER-ORDER ARRAY METHODS ///////////////
 /////////////// HIGHER-ORDER ARRAY METHODS ///////////////
 
+
+/**
+ * ### Challenge `lowerCaseStrings`
```
Comment (Ladrillo) [inline index.js]: I believe this function is not being exposed at the bottom of this file, so the test can't pass

File: index.js [hunk 2]
```diff
@@ -163,6 +139,84 @@ function processDuplicateFree(/* CODE HERE ONLY AFTER COMPLETING ALL OTHER TASKS
...
+ * [1] Invoking `lowerCaseStrings` with `[ 'Orange', 'APPLE', 'banana', 'mAnGo']` will return `[ 'orange', 'apple', 'banana', 'mango' ]`.
+ * 
+ * [2] Invoking `lowerCaseStrings` with `['a', 'b', 'c' ]` will return `[ 'a', 'b', 'c' ]`.
+*/
+function lowerCaseStrings(/* code here */) {
+  /* code here */
+}
+
+/**
+ * ### Challenge `isItAnApple`
```
Comment (Ladrillo) [inline index.js]: Isn't this the same exercise "isItAnApple" from the JS-Exercise-Functions-Arrays-Objects challenge?

File: index.js [hunk 3]
```diff
@@ -163,6 +139,84 @@ function processDuplicateFree(/* CODE HERE ONLY AFTER COMPLETING ALL OTHER TASKS
...
+ * [1] Invoking `isItAnApple` with `[ 'orange', 'apple', 'banana', 'apples', 'apple', 'mango' ]` will return `[ false, true, false, false, true, false ]`.
+ * 
+ * [2] Invoking `isItAnApple` with `['a', 'b', 'c' ]` will return `[ false, false, false ]`.
+*/
+function isItAnApple(/* code here */) {
+  /* code here */
+}
+
+/**
+ * ### Challenge `removeApple`
```
Comment (Ladrillo) [inline index.js]: Same. Dupe from JS-Exercise-Functions-Arrays-Objects ?

File: index.js [hunk 4]
```diff
@@ -236,26 +292,36 @@ function tallyUpDonations(/* CODE HERE */) {
...
+ * 3. In what scenario would the counter1 code be preferable? In what scenario would counter2 be better? 
+ *
 */
+
+// counter1 code
 function counterMaker() {
-  // BROKEN CODE STARTS
-  const count = 0;
+  let count = 0;
   function counter() {
```
Comment (Ladrillo) [inline index.js]: Should we add a return keyword in this line, to return the inner function?

File: index.js (hunk 2 above)
Comment (dryanmas) [reply index.js]: It's the same function, but they're being asked to implement it with .map() this time instead of with a for loop.

Comment (Ladrillo) [reply index.js]: Ohhh, nice

**Pull request merged**